ClientSide(FrontEnd)/clay_/src/components/login.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import './login.css';
+import { useState } from 'react';
+import image4 from '/home/jtbbjoining/Pottery_Marketing/ClientSide(FrontEnd)/clay_/src/Screenshot from 2023-06-06 17-07-16.png';
+function Login({islogin,setIsLogin}){
+    const [username1,setUsername1]=useState("");
+    const [password1,setPassword1]=useState("");
+    
+    function btnClick(){
+       
+        if(username1 =="" ||password1 ==""){
```

Comment:
use prettier code formatter for code format

Reply:
i changed == to ===